packages/crypto/src/networks/mainnet/milestones.json

```diff
@@ -39,5 +39,11 @@
     {
         "height": 8000000,
         "vendorFieldLength": 255
+    },
+    {
+        "height": 8050000,
```

Comment:
The exact milestone has yet TBD but it makes sense to couple it with the `vendorFieldLength`.

Reply:
I decoupled it from the vendorFieldLength on purpose - so that if something breaks we would have a better idea what has caused it.